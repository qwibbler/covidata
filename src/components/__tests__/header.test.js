import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import configureMockStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import Header from '../header';

const mockStore = configureMockStore();
const store = mockStore({
  home: {
    date: '2020-09-09',
    changeDate: false,
  },
});

it('renders correctly', () => {
  const component = renderer.create(
    <Provider store={store}>
      <Router>
        <Header />
      </Router>
    </Provider>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
