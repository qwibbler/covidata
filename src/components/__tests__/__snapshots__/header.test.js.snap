// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header>
  <button
    onClick={[Function]}
    type="button"
  >
    &lt;
  </button>
  2022-02-02
  <button
    type="button"
  >
    âš™
  </button>
</header>
`;
